src/main/resources/feedbackQuestionRubricResultStatsAverageFragment.html

```diff
@@ -0,0 +1,3 @@
+<td>
+    ${average}
+</td>
```

Comment:
Can this be moved somewhere? It looks really really odd to have a template file with just 3 lines.